src/Forms/FormField.php

```diff
@@ -424,8 +424,6 @@ public function getTemplateHelper()
 
     /**
      * Returns the field name.
-     *
-     * @return string
```

Comment:
This change should have been reverted but at this stage I'm sick of asking for them to be reverted so we'll just ignore it lol